### ISSUE TITLE
Uncaught Error: [$injector:unpr] Unknown provider: $$qProvider <- $$q <- $animate <- $compile

### ISSUE THREAD

**Issue by thinq4yourself** [BeardandFedora/writeon.io#12]
After solving last issue, #10, and deplying to Heroku, the app is hosed. 

I notice that it is related to an Angular module injection issue.

> Uncaught Error: [$injector:unpr] Unknown provider: $$qProvider <- $$q <- $animate <- $compile
1. I reinstated `$locationProvider` module and `$locationProvider.html5Mode(true);`, removed during issue #1 resolution
2. Reverted, and still does not work. Besides, `$$qProvider` is not locationProvider. 
3. Tracking down, but see downtime report below.

![screenshot 2015-01-20 at 11 56 44 - display 2](https://cloud.githubusercontent.com/assets/147231/5821613/7bb9fb70-a09b-11e4-86d1-a97721449fbf.png)

**Comment by thinq4yourself** [comment]
Bower updated, bower.json updated. Looking like it may be related to ngAnimate module... or Angular version

## \- https://github.com/angular/bower-angular-animate

**Comment by thinq4yourself** [comment]
When running a  `$ bower update angular`, this is received:

``` text
Please note that,                                                                                                                                             
    ngFitText#3.1.0 depends on angular#1.2.24 which resolved to angular#1.2.24                                                                                
    Writeon.io depends on angular#^1.3.0 which resolved to angular#1.3.9                                                                                      
    angular-bootstrap#0.11.2 depends on angular#>=1 which resolved to angular#1.3.9                                                                           
    angular-animate#1.3.9 depends on angular#1.3.9 which resolved to angular#1.3.9                                                                            
    angular-mailchimp#97fbedff89 depends on angular#~1.3.2 which resolved to angular#1.3.9                                                                    
Resort to using angular#>=1 which resolved to angular#1.2.24                                                                                                  
Code incompatibilities may occur.                         
```

**Comment by thinq4yourself** [comment]
Related to Angular version, ngAnimate, and ngFitText angular version dependencies, which caused code incompatibilities
